app.css

```diff
@@ -36,3 +36,37 @@ body {
 .head-wrapper header ul li:hover {
     border-bottom: 1px solid #4cc5e2;
 }
+
+/*Slick*/
+.single-item {
+  direction: ltr;
+  margin: auto;
+  margin-top:80px;
```

Comment:
Whatever code lines you add in this PR are part of it. Added lines are marked in green. If you want something to be part of a different PR don't add them here